openmdao/surrogate_models/kriging.py

```diff
@@ -150,6 +156,14 @@ def _calculate_reduced_likelihood_params(self, thetas=None):
         params['U'] = U
         params['Vh'] = Vh
 
+        # Newly added for EGOLF
+        one = np.ones([self.n_samples,1])
+        R_inv = Vh.T.dot(np.dot(np.diag(inv_factors),U.T)) #Use einsum in the future release for efficiency
+        params['R'] = R
+        params['R_inv'] = R_inv
+        params['mu'] = np.mean(self.Y,axis=0)
```

Comment:
I would like to have design variables normalized to [0,1] space. But there is no problem so far running EGOLF by normalizing this way.